src/graph.js

```diff
@@ -44,15 +44,52 @@ class Graph {
 			strings.push(resultSet.next().getString(0));
 		}
 		return strings;
-	}
+    }
+    
+    paramToString(paramValue) {
+        if(paramValue == null)
+            return "null"
+        let paramType = typeof(paramValue);
+        if(paramType == "string") {
+            let strValue = "";
+            if (paramValue[0] != "\"")
+                strValue += "\"";
+            strValue += paramValue;
+            if(paramValue[paramValue.length-1] != "\"")
+                strValue += "\"";
+            return strValue;
+        }
+        if(Array.isArray(paramValue))
+            return "[" + paramValue.toString() + "]";
```

Comment:
should every paramValue be checked as String and escaped?

Reply:
added test